www/source/images/icons/icon-slack-white.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
These social icons are currently being represented with a PNG sprite and accompanying CSS. I like that we're replacing them with SVGs, but could we also remove the dead CSS code and `icons-social` images?

Reply:
Yeah, I mean honestly, if it were me, I'd clean it out for sure. It's just good practice not to leave unused stuff around when you're aware of it.